### PR TITLE
Build, archive, and provide -dbgsym packages

### PR DESCRIPTION
**Pull request by peat-psuwit**
This PR enables building & using .ddeb packages, which is useful when debugging software.

### CODE REVIEW

File: create_repository.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ apt-ftparchive sources . \
     | tee "$where"/Sources \
     | gzip -9 > "$where"/Sources.gz
 
-apt-ftparchive packages "$where" \
+apt-ftparchive -o 'Packages::Extenstions=.deb,.ddeb' packages "$where" \
```
Comment (mardy) [inline create_repository.sh]: Did you test this? `Extenstions` doesn't sound correct.

Comment (peat-psuwit) [reply create_repository.sh]: Yes, I've tested it. The option is specified in the [apt-ftparchive manpage](https://manpages.debian.org/jessie/apt-utils/apt-ftparchive.1.en.html).

Comment (mardy) [reply create_repository.sh]: But there it's `Packages::Extensions`, not `Packages::Extenstions`...

Comment (peat-psuwit) [reply create_repository.sh]: Oops, I missed that. 🤦

However, turns out the option isn't required for it to include .ddeb packages (That's why it works despite getting the option name wrong). I'll rebase while removing the create_repository.sh part and re-submit it.

**Comment by mardy** [comment]
Thanks!

**Pull request merged**